ocis/pkg/runtime/service/service.go

```diff
@@ -93,6 +95,7 @@ func NewService(options ...Option) (*Service, error) {
 		cfg:          opts.Config,
 	}
 
+	s.ServicesRegistry["audit"] = audit.NewSutureService
```

Comment:
do we really wan to start it as a default service with the fullstack server? I'm not sure if it provides that much value on a raspberry pi

Reply:
Uh. You are right. And it is also written in the Jira-Ticket:
```
the audit service is default off (not part of the fullstack server start command)
```
Looks like I should practice reading. Thanks for pointing it out.

Reply:
removed it